stock_production_lot_active/models/stock_production_lot.py

```diff
@@ -1,4 +1,3 @@
-# Copyright 2020 ACSONE SA/NV
```

Comment:
Copyrights on Python files surely should not be deleted.

Reply:
@janikvonrotz better to keep the file untouched to reduce the diff to the bare minimum, the copyright was on the first line